examples/server.py

```diff
@@ -27,7 +27,8 @@ def parse_args():
     ''' Parse command line arguments. '''
     parser = argparse.ArgumentParser(description='Example trio-websocket client')
     parser.add_argument('--ssl', action='store_true', help='Use SSL')
-    parser.add_argument('ip', help='IP to bind to')
+    parser.add_argument('host', help='Host interface to bind to (* binds to all'
```

Comment:
Are you intending that not specifying host is the same as *?  I think that's the case.

Reply:
Host is a required argument, so I needed a special string to indicate "bind all". It could be a keyword argument instead, but most programs default the host interface to either `0.0.0.0` or `localhost`, which might make this a surprising behavior. I messed around with `netcat` and it doesn't seem to have any way to bind all interfaces. By default it binds `0.0.0.0`, and with the `-6` flag it binds `::`. I think netcat doesn't bind to IPv4 and IPv6 at the same time because the C API doesn't make it easy to do so?

Maybe there's a better string than `*`, which is annoying to use in most Linux shells (have to write `python server.py '*' 8000`). Or since this is the example server, is it not worth the effort to provide this feature?

Reply:
I don't have an issue with `*`, it's that the `host` default is `None`, which will be equivalent to `*` due to an implementation detail.  So perhaps disallow `None` and default host to `*`.

Reply:
no, I mean that the parser arg is missing a default.  The arg's default should probably be `*`.  Without a default, args.host will be `None`,  and when passed to the library `None` will be treated as listen all.  So there are two ways on the command line to get the same result...

Reply:
Ohhh, okay, I didn't realize that argparse supports optional positional parameters. I will remove `*` and just use `None`: 

```$ python examples/server.py 8000
INFO:root:Starting websocket server…
DEBUG:trio-websocket:Listening on ws://0.0.0.0:8000,ws://[::]:8000```

Reply:
oh my mistake!  I wasn't paying attention that it was a positional param.  Should be ok as is.  Not enough sleep-- sorry.